### ISSUE TITLE
GitHub Windows overwrites global config Git Extensions created

### ISSUE THREAD

**Issue by mccreeker** [gitextensions/gitextensions#4203]
Not sure if this is the proper place to put this or if the Git Extensions team can do anything about it, but my usage of it is being interrupted by other Git software, and I'd like to use GitHub Desktop for github.com repos, SourceTree for Bitbucket repos, and Git Extensions for anything else or just whatever.

I'd think the global config that Git Extensions uses could or would be isolated to its own usage, but maybe it's unavoidable to be accessed and modified by multiple installed programs due to the way environment variables or Windows user accounts work etc. I would love to see an option to isolate Git Extensions user/email defaults, as I've already run into problems with local repo info not saving after being closed in the app.

**Do you want to request a *feature* or report a *bug*?**
Bug?

**What is the current behavior?**
Installing GitHub Windows uses the same global config Git Extensions does.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
Install and config Git Extensions, clone a repo with your user/email/whatever, then install GitHub Desktop and clone a repo from GitHub; notice your global settings have been changed by GitHub Desktop.

**What is the expected behavior?**
Git Extensions global config shouldn't be messed with by other apps, maybe it can be installed in its own folder.

**Environment you encounter the issue:**
Windows 7, Windows 10

- GitExtensions version: 2.5
- GIT version: 2.15
- OS version: Pro

**Did this work in previous version of GitExtensions (which)?**
It's always been mucked with by other apps.

**Comment by gerhardol** [comment]
Setting HOME in GE config should give you what you want

**Comment by vbjay** [comment]
You will still have issues with git locking if you have the same repo open
in two tools. FyI, we have a GitHub plug-in that let's you create pull
requests.

**Comment by mccreeker** [comment]
I'll try changing HOME but it seemed to have issues with that so I'm guessing I have to move the .git folder from my user folder to another one before changing the setting.

I don't open the same repo in different git GUIs. The point was to use GitHub Desktop for GitHub repos, SourceTree for Bitbucket repos, and Git Extensions for other repos (or use it instead of SourceTree for certain repos).

**Comment by vbjay** [comment]
Use whatever you want.  We support GitHub pull requests, all of those git
servers are just remotes. You can use git extensions for all or some.

**Comment by mccreeker** [comment]
I realize Git Extensions supports GitHub, that's also not the point. There's just little things referenced or utilized with GitHub's tools (like Desktop) that are just nice to sync with their site features (I'm not going to get into it, that's also not the point), including with SourceTree and Bitbucket.

The point of this issue was to just figure out if there was anything I could do with Git Extensions to prevent collisions like I described with other installed git tools, or just learn a better way to use it in this regard, which I discovered here, so thank you, and I'll close the issue.
